index.js

```diff
@@ -0,0 +1,66 @@
+const baseURL = 'https://api.ratesapi.io/api/';
+const summaryURL = 'latest';
+const rateURL = 'latest?base=USD&symbols=GBP';
+
+const baseElement = document.getElementById('base');
+const symbolElement = document.getElementById('symbol');
+const rateElement = document.getElementById('rate');
+
+const fetchSymbols = () => {
+  return fetch(baseURL + summaryURL)
+    .then((response) => {
+      if (response.status === 200) {
+        return response.json();
+      }
+      throw new Error(response.statusText);
+    });
+}
+
+const fetchRate = (base, symbol) => {
+  return fetch(`${baseURL}latest?base=${base}&symbols=${symbol}`)
```

Comment:
I think you can create one function for `fetchSymbols` and `fetchRate` because differences are only in params